src/oisst/Geometry/Geometry.cc

```diff
@@ -9,42 +9,59 @@
 
 #include "eckit/config/Configuration.h"
 
+#include "atlas/grid.h"
+
 #include "oops/util/abor1_cpp.h"
 
 namespace oisst {
 
 // ----------------------------------------------------------------------------
 
-  Geometry::Geometry(const eckit::Configuration & conf,
-                     const eckit::mpi::Comm & comm)
-    : comm_(comm) {
-    util::abor1_cpp("Geometry::Geometry() needs to be implemented.",
-                    __FILE__, __LINE__);
+  Geometry::Geometry(const eckit::Configuration & conf, const eckit::mpi::Comm & comm) : comm_(comm) {
+  //util::abor1_cpp("Geometry::Geometry() needs to be implemented.", __FILE__, __LINE__);
+    
+    atlas::RegularLonLatGrid atlasRllGrid("Slat360x180");
```

Comment:
If you wanted to go ahead and move this into the configuration file with this PR, i think the following would work
```suggestion
    atlas::RegularLonLatGrid atlasRllGrid(conf.getString("grid"));
```
then in your `geometry.yaml`
```
geometry:
  grid: Slat360x180
```

Reply:
Done.